assets/composable/containerActions.ts

```diff
@@ -0,0 +1,47 @@
+type ContainerActions = "start" | "stop" | "restart";
+export const useContainerActions = () => {
+  const context = inject(containerContext);
```

Comment:
If you search the code a little bit, you see `const { container } = useContainerContext();` is the right thing to use.